core/dbt/events/functions.py

```diff
@@ -223,7 +237,11 @@ def fire_event(e: Event) -> None:
 
     # always logs debug level regardless of user input
     if isinstance(e, File):
-        log_line = create_json_log_line(e, msg_fn=lambda x: x.file_msg())
+        log_line = (
+            create_json_log_line(e, msg_fn=lambda x: x.file_msg())
+            if this.format_json else
+            create_text_log_line(e, msg_fn=lambda x: x.file_msg())
+        )
```

Comment:
let's encapsulate this into a function since it's used 3 times.
```suggestion
        log_line = create_log_line(json=this.format_json, e=e, msg_fn=lambda x: x.file_msg())
```
where
```
def create_log_line(json: bool, e: T_Event, msg_fn: Callable[[T_Event], str]) -> str:
    return (
        create_json_log_line(e, msg_fn)
        if this.format_json else
        create_text_log_line(e, msg_fn)
    )
```